include/manif/impl/se_2_3/SE_2_3Tangent_base.h

```diff
@@ -390,8 +390,8 @@ struct RandomEvaluatorImpl<SE_2_3TangentBase<Derived>>
 {
   static void run(SE_2_3TangentBase<Derived>& m)
   {
-    m.coeffs().setRandom();                // in [-1,1]
-    m.coeffs().template segment<3>(3) *= MANIF_PI; // in [-PI,PI]
+    m.coeffs().setRandom(); // in [-1,1]
+    m.coeffs().template segment<3>(3) = randPointInBall(MANIF_PI); // in [-PI,PI]
```

Comment:
Fix comment to `// In ball of radius PI`

Reply:
Done :+1: